### PR TITLE
Apple page cloning

### PR DESCRIPTION
**Pull request by beyk**
This project consists of making a clone of Apple page. 
-Set up the main background image. 
-Set up the top navigation bar. 
-using gradient background -...

### CODE REVIEW

**Comment by adaorachi** [comment]
👋 Hi @beyk,

Well done on the completion of the first draft of this project. You did a really nice job on it. 
The clone shows a good understanding and usage of background images and gradients. 💯 
The cloned page is rendering as it should and meets almost all the project specifications.
Your images and contents are well-positioned. 
Pull Request has a meaningful title and a short description, and commit messages are explanatory.
And your CSS codes look clean as well. 
You have no linter errors at this point.
Goob Job! 👍 

**However, there are just a couple of changes you will need to implement to get an approval for this project. 
So hang in there, you are just a step away from completing this project. 💪** 

---
### **CHANGES REQUIRED** ♻️ 
---

- [ ] First, your repo contains files that are not related to the project and which should not be committed. Your node-module folder should not be added to your commits. Create a .gitignore file in the root folder of your project and include node-module as a list of folders/files to ignore while making commits.
 
---

- [ ] Your README file looks okay but it can be improved on to make it look more professional. The summary/description should be very explanatory showing a few sentences or paragraphs of what you implemented in the project, what you enjoyed doing in the project, and all about the project. In the screenshot below, you have it somewhat like a list. Also, ensure to correct the typo error in the title text.

**YOUR REPO:**

image1

---

- [ ] Your HTML markup has a couple of errors. Check them using the [validator tool ](https://validator.w3.org/#validate_by_upload)  and fix those errors.

---

- [ ] In the screenshot of your page attached below, you'll notice that the last image is cut off. The images do not share an equal width from the parent container. Make sure to modify your CSS styling for those elements to contain all contents perfectly.

**YOUR PROJECT VERSION:**

image2

---

- [ ] You have an attribute called **dideo-checked="true"** in your navigation anchor links markup. Those are not appropriate HTML attribute and should not be used. Kindly remove them. 

---

The requirement below is **NOT CONSIDERED** to be mandatory. But I strongly encourage you to take a moment to look into it as they may come in handy in future projects.

- [**optional**]  You should consider adding a hover effect when the navigation links are hovered on. And also consider removing the text-underline default styling for your anchor tags. This is noticed in the footer texts. These implementations will improve your project aesthetic look.

---

**NOTE: All requirements not marked with [optional] are mandatory!**
### **After implementing the MANDATORY changes highlighted above, request for another review by submitting a Code Review Form.** 

Should you have any questions or need any help, please feel free to reach out to me on Slack(@adaorachi). ☺️ 

**Happy Coding and Good-luck on it.**

**MaryAnn Chukwuka**

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @beyk, you did a great job with this project :+1: However, before we proceed, we have to make some changes to our project so we are able to meet all the minimum requirements. Kindly find comments below:
- Since this project focuses on using ```backgrounds/gradients```, in this case, our focus is on background, kindly remove all the embedded images and replace then with a ```<div>```, give it a class and set the background-image in the CSS file
re1
- The last image is losing some of the widths, we currently do not see the full width. A quick fix
would be to set the wrap to 100% and set each of the divs to 25%
re
- Let us apply the DRY rule. This [article](https://speckyboy.com/good-bad-css-practices/) would help
- Let us avoid using tag names for styling, let use classes only

Kindly implement these changes and submit another code review request :smile:
_Happy coding_ :man_technologist: 
Daniel Larbi Addo on slack

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+linters:
+  stylelint:
```
Comment (lucasmazo32) [inline .stickler.yml]: Please modify this file to be the same as the one provided by Microverse, you can use this [link](https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml).

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (lucasmazo32) [inline .gitignore:1]: Please modify this file, so you include neither the node modules nor the package-lock.json.

Your file can look like this:

```
node_modules
package-lock.json
```

**Comment by lucasmazo32** [comment]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. For the project to get approved, kindly implement the changes that I commented (regarding the stickler and .gitignore file).

After doing so, please submit for another code review 🥇.

### Happy coding! :computer:

## Lucas Mazo

File: assets/styles/main.css [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+  display: flex;
+}
+
+.flex-v {
+  flex-direction: column;
+}
+
+.flex-h {
+  display: flex;
+  flex-direction: row;
```
Comment (santiago-dev96) [inline assets/styles/main.css:29]: Flex boxes have a row direction by default
```suggestion
```

**Review by santiago-dev96 — APPROVED** [review]
# Your project has been approved!

You fulfill all the requirements to pass. I noticed that you have the folder `node_modules` tracked for pushing, in order to delete it from the tracked files (because you already added it in previous commits) you need to issue the following `git` command: `git rm -r --cached node_modules`. Have a nice day!

File: assets/styles/main.css (hunk 3 above)
Comment (beyk) [reply assets/styles/main.css:29]: Thank you for your attention, I fix all issues

**Pull request merged**